cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -347,22 +448,25 @@ def measure_zernike(self, pixels, labels, indexes, centers, radius, n, m):
         # I'll try to explain some - hopefully, you can reuse.
         center_x = centers[:, 1]
         center_y = centers[:, 0]
+
         #
         # Make up fake values for 0 (the background). This lets us do our
         # indexing tricks. Really, we're going to ignore the background,
         # but we want to do the indexing without ignoring the background
         # because that's easier.
         #
-        center_x = np.hstack([[0], center_x])
-        center_y = np.hstack([[0], center_y])
-        radius = np.hstack([[1], radius])
+        center_x = numpy.hstack([[0], center_x])
+        center_y = numpy.hstack([[0], center_y])
+        radius = numpy.hstack([[1], radius])
+
         #
         # Now get one array that's the y coordinate of each pixel and one
         # that's the x coordinate. This might look stupid and wasteful,
         # but these "arrays" are never actually realized and made into
         # real memory.
         #
-        y, x = np.mgrid[0:labels.shape[0], 0:labels.shape[1]]
+        y, x = numpy.mgrid[0.0:labels.shape[0], 0.0:labels.shape[1]]
```

Comment:
Why the `0.0`?

Reply:
Fun trick: it makes the data have floating-point values, rather than integers. It's needed for a `+=`'s below, which requires matching datatypes.

Reply:
I've added a comment explaining this.